engine/build.gradle

```diff
@@ -244,3 +244,8 @@ idea {
         downloadSources = true
     }
 }
+
+// Make sure our config file for code analytics get extracted (vulnerability: non-IDE execution of single analytic)
+ideaModule.dependsOn rootProject.extractConfig
+tasks.eclipse.dependsOn rootProject.extractConfig
+check.dependsOn rootProject.extractConfig
```

Comment:
I believe that this could fail randomly, since it does not guarantee that `extractConfig` is run before the checkstyle tasks. This only ensures that it's run before the (no-op) task `check`.
I assume that the solution that was posted on the gradle forums is the very slow one?